portfolio.css

```diff
@@ -0,0 +1,245 @@
+*{
+	box-sizing: border-box;
+	color:#fff;
+}
+
+/* Background image css */
+
+body{
+	background-image: url(mountain_bg.jpg);
+	background-repeat: no-repeat;
+	background-position: bottom;
+	background-color:#FBEEC1;
+	background-attachment:fixed;
+	background-size:cover;
+}
+
+/* Header formatting */
+
+h1, h2{
+	background-color:#115464;
+	border-radius:16px;
+	border-style: solid;
+	border-width:2px;
+	border-color:#0c96b0;
+	margin:auto;
+	margin-top:10px;
+	margin-bottom:10px;
+	padding:5px;
+	font-size: 30px;
+}
+
+.projects h1{
+	margin-bottom:5px;
+}
+
+h3{
+	display:inline;
+	background-color:#f7ebae;
+	border-radius:16px;
+	border-style: solid;
+	border-width: 2px;
+	border-color:#228db1;
+	margin: 10px auto;
+	padding:5px;
+	font-size: 30px;
+	color:#000;
+}
+
+/* Hyperlink formatting */
+
+a, a:active{
+	font-size:25px;
+	padding:5px;
+	border-radius:16px;
+	border-style: solid;
```

Comment:
Very picky but maybe try and stay consistent with spaces/no spaces after your colons throughout